applications/community-health-check.md

```diff
@@ -0,0 +1,281 @@
+
+- **Total Estimated Duration:**  10 weeks
+- **Full-Time Equivalent (FTE):**  6
+- **Total Costs:**  15 000 USDC
+
+We are following a grant-matching processes to spread the risk among different communities. 
+
+### Milestone 1 - Defining metrics
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1
+- **Costs:** 5000 USDC
+
+Research social network metrics that are viable to measure the health of a community on Twitter, and how they integrate with our existing community health framework. This will build on our existing framework, extending it to metrics that have been tested and used in Twitter communities. For example, we will look into approaches to build the network, virality and clustering/fragmentation of very large online communities. 
+
+Note: We have done an evaluation to decide wheter Reddit or Twitter would be a better option. Given other requests we have received, we have decided that building an integration and health metrics for Twitter is a more viable path for us.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The results will be published open-access (no paywall or copyright). |
+| 0b. | Documentation | No documentation necessary.|
+| 0c. | Testing and Testing Guide | Testing and Testing Guide not necessary. |
+| 0d. | Docker | A docker file is not necessary for using the community health check. |
+| 0e. | Article| We will publish an article (technical document) describing the metrics, the insights (so-what), and limitations. This article will also explain how the Twitter community is build. |
+
+### Milestone 2 — Twitter community health dashboard
+
+- **Estimated duration:** 6 weeks
+- **FTE:**  5
+- **Costs:** 10 000 USDC
+
+This milestone implements the work from the previous milestone by building the dashboard.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The code for data fetching and storage aisre already available in a public github folder (Unlicensed). We will specify an OSS license. |
+| 0b. | Documentation | We will provide a tutorial for users to understand how to run the health checks themselves. Our current version will be updated to include recent development.|
+| 0c. | Testing and Testing Guide | Running the data pipeline and analyzing the data will be covered by tests to ensure functionality. We will describe how to run the tests|
+| 0d. | Docker | We will run a docker file to tests the functionality. |
+| 0e. | Article | In addition to the dashboard, we write a handout. This is a walkthrough of the dashboard, explaining each metric, if the score is good or bad, and a list of recommendations.  |
+| 1. | Twitter data integration | We will create a data pipeline fetching data from Twitter using their API. The user will have several options to define their community (e.g., everyone who follows the official Polkadot/Kusama account, everyone who follows the account and interacts with the content). |
```

Comment:
- Will you write the integration for a paid version of the API or the free tier?
- Can you elaborate on the throughput that this your crawler will be able to deliver (e.g. how many requests/min, how many tweets, comments, likes, etc. can be retrieved by some unit of time).
- How about other accounts, e.g. parachains, dApp developers and other teams that build on Substrate?

Reply:
The updated proposal will have more information.

Reply:
Similarly to the research deliverable in M1, can you please add more specific information about which edges will be supported?

Reply:
I have decided to not repeat the same information here than for Milestone 1. Milestone 1 is our research phase, while milestone 2 is the implementation. Hence, decisions we are making during milestone 1 will impact milestone 2.

Reply:
@katerinabc lovely, thanks for adding that info.